ckanext/opendata_theme/opengov_custom_header/plugin.py

```diff
@@ -0,0 +1,141 @@
+import re
+import six
+import string
+
+from ckan.logic.validators import Invalid
+from six.moves.urllib.parse import urlparse, quote
+import ckan.plugins as plugins
+import ckan.plugins.toolkit as toolkit
+from ckan.common import config
+from ckanext.opendata_theme.opengov_custom_header.controller import CustomHeaderController, Header
+from webhelpers.html import escape, HTML, literal, url_escape
+
+from ckanext.opendata_theme.opengov_custom_header.constants import CONFIG_SECTION, CONTROLLER, DEFAULT_CONFIG_SECTION
+
+
+class Opendata_ThemePlugin(plugins.SingletonPlugin):
+    plugins.implements(plugins.IConfigurable, inherit=True)
+    plugins.implements(plugins.IConfigurer)
+    plugins.implements(plugins.ITemplateHelpers)
+    plugins.implements(plugins.IRoutes, inherit=True)
+    plugins.implements(plugins.IValidators)
+
+    def get_validators(self):
+        return {
+            u'custom_header_url_validator': custom_header_url_validator,
+        }
+    # IConfigurer
+    def update_config(self, ckan_config):
+        toolkit.add_template_directory(ckan_config, 'templates')
+        toolkit.add_public_directory(ckan_config, 'static')
+        toolkit.add_resource('../base/fanstatic', 'opengov_custom_theme_resource')
+        toolkit.add_resource('../opengov_custom_header/fanstatic', 'opengov_custom_header_resource')
```

Comment:
I don't see a fanstatic directory in `ckanext/opendata_theme/opengov_custom_header`.

Reply:
removed